src/renderer/App.tsx

```diff
@@ -175,55 +175,9 @@ function Sidebar() {
 
 function Topbar() {
   return (
-    <Navbar sticky="top" bg="primary" variant="dark" expand="sm">
```

Comment:
Please don't pull the top bar out - we need to have some similarity in look to the web version - and there, not having a top is really weird.

Reply:
Can you tell me more about what's weird about it? To me, it just takes up precious vertical screen real estate and doesn't add too much, since we could just have a logo in the top left hand corner which would accomplish approx. the same. Maybe a good compromise would be to make it smaller?

Reply:
Is the goal to have it anchor (heh) the navigation so that it serves a purpose similar to what "Events Explorer" does here?
<img width="920" alt="Screen Shot 2022-04-02 at 9 03 25 AM" src="https://user-images.githubusercontent.com/1476820/161391345-cf1741a1-e787-4731-a11d-9ff1af49029d.png">

Reply:
so, weird - its a combination of "norms", and if you're going to have the logo and the network selector, then you already _have_ a top bar. I also don't like unlabeled icons - they're not "self explanatory" - especially as we're reusing iconography that has quite different meaning.

but yup, very much like the events explorer pinning.